src/main/resources/templates/progress/getHintList.html

```diff
@@ -18,6 +18,7 @@ <h2>Odemknuté nápovědy</h2>
                     <p>
                         <span class="hint-label" th:text="|Nápověda (-${hintTaken.hint.value}b.):|"></span>
                         <span th:text="${hintTaken.hint.text}"></span>
+                        <img th:if="${hintTaken.hint.imageUrl}" alt="Obrázek nápovědy" src="" th:src="@{${hintTaken.hint.imageUrl}}" />
```

Comment:
Urcite pridat `class="img-fluid"` a mozna to dat az za ten odstravec? Ale musel bych zkusit jak se to zobrazuje, tak to necham na tvem uvazeni ;)